crane-cloud/frontend
# feat: introduce app category icons to distinguish regular apps from AI application deployments

# Description

- This PR introduces icons to the app card to create distinction for regular application deployments from AI deployments.
- Also ensures the app cards occupy the entire width of each row!

## Type of change


## Clickup Ticket ID

https://app.clickup.com/t/8695tqc03

## How Can This Been Tested?

- Run this branch locally and visit any of your projects that has various application deployments [regular or AI] to notice the icons.

## Checklist:


# Screenshots

![Screenshot 2024-09-26 at 4 53 34 PM](https://github.com/user-attachments/assets/2c9fffb3-8034-435f-8079-a369f0b00d5d)

![Screenshot 2024-09-26 at 4 52 21 PM](https://github.com/user-attachments/assets/e02e4815-e88f-4b9a-8230-cdde792968b1)
